Examples/cChocoFeatureExample.ps1

```diff
@@ -0,0 +1,28 @@
+
+
+configuration ChocoFeatures {
+
+    Import-DscResource -ModuleName cChoco
+
+    Node 'localhost' {
+
+        cChocoFeature allowGlobalConfirmation {
+
+            FeatureName = "allowGlobalConfirmation"
```

Comment:
No, not to my knowledge - I think it's a problem with the DSL for DSC no way of indicating this in syntax and no way of extending it. 

Even if the resource name could be queried at the module level, the configuration wouldn't ever compile without a the FeatureName key